gtda/homology/simplicial.py

```diff
@@ -445,14 +449,14 @@ def transform(self, X, y=None):
 
 @adapt_fit_transform_docs
 class EuclideanCechPersistence(BaseEstimator, TransformerMixin):
-    """`Persistence diagrams <https://giotto.ai/theory>`_ resulting from
-    `Cech filtrations <https://giotto.ai/theory>`_.
-
-    Given a `point cloud <https://giotto.ai/theory>`_ in Euclidean space,
-    information about the appearance and disappearance of topological
-    features (technically, `homology classes <https://giotto.ai/theory>`_) of
-    various dimensions and at different scales is summarised in the
-    corresponding persistence diagram.
+    """:ref:`Persistence diagrams <persistence diagram>` resulting from
+    `Cech filtrations <>`_.
```

Comment:
Just linking this to #295 and #295.

Reply:
Did you mean to link the same issue twice?

Reply:
One is an issue, the other a PR. Maybe not what you meant?

Reply:
Note to self: read things: The second thing should have been #336